post-script.rhai

```diff
@@ -0,0 +1,3 @@
+if variable::get("mcu") != "esp32h2" && variable::get("wifi"){
```

Comment:
```suggestion
if variable::get("wifi"){
```
I think this can be simplified to just this because the default is false and on h2 the question won't get asked?

Reply:
Not sure, I guess my rhai knowledge is not good enough :D. If the previous check worked then we can revert to that.

Reply:
Reverted!